packages/frontend/src/components/SimpleChart.tsx

```diff
@@ -83,6 +83,17 @@ export const SimpleChart: FC<SimpleChartProps> = ({
     chartType,
     chartConfig,
 }) => {
+    useEffect(() => {
+        if (!chartRef.current) return;
+
+        const listener = () => {
+            const eCharts = chartRef.current?.getEchartsInstance();
+            eCharts?.resize();
+        };
+
+        window.addEventListener('resize', listener);
```

Comment:
We should remove the event listener when the hook "unmounts".
Example here: https://usehooks.com/useEventListener/